unlock-js/src/constants.js

```diff
@@ -1,9 +1,11 @@
 import { ethers } from 'ethers'
 
+// TODO: gas prices should be custom per version!
```

Comment:
Are these prices or limits?

Reply:
you are right, they are not prices, but amounts of gas to be spent. I'll fix the comment